### PR TITLE
Query And Update Animals Table

### PR DESCRIPTION
**Pull request by DammyShittu**
In this milestone, I;

- added a new column (species) to the `animals` table,

- inserted six(6) new rows into the `animals` table,

- used database transactions to `update` and `delete` table data,

- used `aggregates` with `GROUP BY` statements to prepare complex queries that answer analytical questions

- wrote queries to answer six(6) questions,

- added SQL code to the appropriate files(`schema.sql`, `data.sql`, `queries.sql`),

- added the MIT license file, and 

- personalized the README file.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -24,4 +22,54 @@ SELECT * FROM animals WHERE neutered = true;
...
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Two
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE NOT name LIKE '%mon';
```
Comment (Emmyn5600) [inline queries.sql]: - [ ] `UPDATE` the animal's table by setting the species column to pokemon for all animals that don't have species already set. however, you are updating the species column to pokemon for all animals that have a name ending on mon.

File: queries.sql [hunk 2]
```diff
@@ -24,4 +22,54 @@ SELECT * FROM animals WHERE neutered = true;
...
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO point1;
+COMMIT;
+
+
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(escape_attempts) FROM animals GROUP BY escape_attempts HAVING escape_attempts = 0;
```
Comment (Emmyn5600) [inline queries.sql]: - [ ] Optional
Here you are trying to display the animals that have never tried to escape, so the condition is very simple, you can change the logic here.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi,

Good job so far! 👏🏼 @DammyShittu 

## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- _Check the comment under the review._
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: queries.sql (hunk 2 above)
Comment (DammyShittu) [reply queries.sql]: Thank you @Emmyn5600,
I will work on this optional suggestion now.

File: queries.sql (hunk 1 above)
Comment (DammyShittu) [reply queries.sql]: Hello @Emmyn5600,
Thank you for this. I'll work on it now.

**Comment by DammyShittu** [comment]
Hey @Emmyn5600,
I have worked on the requested changes and I will proceed to request another review now.
Thanks 👍

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @DammyShittu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DammyShittu** [comment]
Thanks, @DeliceLydia 🎉

**Pull request merged**